randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1110,8 +1110,21 @@ Extra - polygon: [[-900.0, 3700.0], [-5000.0, 3700.0], [-5000.0, -2115.0], [-900
       Trivial
   > Pickup (Energy Tank)
       All of the following:
-          Flash Shift and Morph Ball
-          Grapple Beam or Spider Magnet or Walljump (Beginner)
+          # You always need morph
+          Morph Ball
+          Any of the following:
+              # Either flash or grapple with a trivial wall jump, or space jump directly
+              Flash Shift or Grapple Beam or Space Jump
+              All of the following:
+                  # Speed boost into spider magnet surface or beginner walljump
+                  Speed Booster
+                  Spider Magnet or Walljump (Beginner)
+              All of the following:
+                  # Midair sideways cross bomb into spider magnet surface or onto frozen enemy
+                  Lay Cross Comb
+                  Any of the following:
+                      Spider Magnet
+                      Stand on Frozen Enemy (Intermediate) and Shoot Ice Missile
```

Comment:
i still want to say you can probably walljump after the cross bomb without using the enemy but would need tested. this should also probably include Damage Boost (Beginner).

Also, Spin Boost alone is also enough to get into the hole but it's rather difficult. I'd place it at Movement (Intermediate)

Reply:
I did this. It's unclear if the damage boost or spin boost should then also be paired with a walljump.